diesel/Cargo.toml

```diff
@@ -31,6 +31,7 @@ tempdir = "^0.3.4"
 
 [features]
 default = ["with-deprecated"]
+all = ["chrono", "quickcheck", "serde_json", "uuid"]
```

Comment:
If anyone were using `PgDateTime`, `PgNumeric` and friends or the interval DSL and wanted to test stuff involving it, yes. But I agree it doesn't warrant inclusion here.